#!/usr/bin/env python2
from dynamic_reconfigure.parameter_generator_catkin import *
PACKAGE = "asr_ism_visualizations"


gen = ParameterGenerator()

gen.add("objectAxisMap", str_t, 0, "Axis of object to visualize for valid position visualization.", "Object1-Axis1;Object2-Axis2")
gen.add("objectSampels", int_t, 0, "Samplerate of objects for valid position visualization.", 5, 1, 10)
gen.add("posePredictionAlpha", double_t, 0, "Alpha of Poseprediction for valid position visualization.", 0.05, 0, 1)
gen.add("posePredictionValue", double_t, 0, "Value of Poseprediction for valid position visualization.", 0.8, 0, 1)
gen.add("sampleAlpha", double_t, 0, "Alpha of objectsamples for valid position visualization.", 1, 0, 1)
gen.add("sampleValue", double_t, 0, "Value of objectsamples for valid position visualization.", 1, 0, 1)
gen.add("sampelScale", double_t, 0, "Scale of objectsamples for valid position visualization.", 0.1, 0, 1)

exit(gen.generate(PACKAGE, "asr_ism_visualizations", "ism_valid_position_visualizer"))
